Doc/library/datetime.rst

```diff
@@ -589,8 +589,8 @@ Supported operations:
 +-------------------------------+----------------------------------------------+
 | Operation                     | Result                                       |
 +===============================+==============================================+
-| ``date2 = date1 + timedelta`` | *date2* is ``timedelta.days`` days removed   |
-|                               | from *date1*. (1)                            |
+| ``date2 = date1 + timedelta`` | *date1* is ``timedelta.days`` days removed   |
```

Comment:
The result should be what will be the value of `date2` because `date2` is what being assigned to. Suggestion:

```Result is `timedelta` added to `date1` 
```

Even though it seems a bit repetitive to restate the operation on the left, it's already very intuitive and it would confirm to the reader that the `+` operator here does the obvious thing and not something weird.

Reply:
How about ```*date2* will be ``timedelta.days`` after *date1*.``` I think "after" indicates the direction correctly.

Reply:
That sounds good to me, using "after" here makes sense.